lib/api/controllers/securityController.js

```diff
@@ -550,3 +564,57 @@ function resetProfiles () {
     .then(() => this.internalEngine.createOrReplace('profiles', 'default', {policies: [{roleId: 'default'}]}));
 }
 
+/**
+ *
+ * @param {Kuzzle} kuzzle
+ * @param {string.<profile|role|user>} type
+ * @param {Request} request
+ * @returns {Promise.<*>}
+ */
+function mDelete (kuzzle, type, request) {
+  const
+    action = _.upperFirst(type);
+
+  assertHasBody(request, `security:${action}`);
+  assertBodyHasAttribute(request, 'ids', `security:${action}`);
+  if (!Array.isArray(request.input.body.ids)) {
```

Comment:
nitpicking: replace it by `assertHasIndexAndCollection` introduced here: https://github.com/kuzzleio/kuzzle/pull/616/files/eb4baa72454df15e01cc11bdc246029bbd5e52e2..98450af8137849d5e3b0942b949ab64e248c765b#diff-91a156dc5237febe996201236bc18f66R44

(implies a dependency with #616)